docs/dev-guide/newtypesupport/plugin/migrations.rst

```diff
@@ -10,7 +10,9 @@ Registration
 ============
 
 In order to write migrations for your Pulp plugin, you will need to register your plugin's
-migrations package with the Pulp server.
+migrations package with the Pulp server. It is advantageous to register even a brand-new plugin
```

Comment:
It might be nice to put a similar note somewhere that a new plugin writer is guaranteed to see it. The intended audience of this particular note might not have a reason to look here until they need to write a migration for a plugin that is not brand-new.